test/libsolidity/syntaxTests/nameAndTypeResolution/array_length_fractional_computed.sol

```diff
@@ -0,0 +1,10 @@
+contract test {
+    uint constant a = 7;
+    uint constant b = 3;
+    function f() public {
+        uint[a / b] memory x; x[0];
+        uint[7 / 3] memory y; y[0];
+    }
+}
+// ----
+// TypeError 3208: (141-146): Array with fractional length specified.
```

Comment:
This is one error message for two error cases (they're fatal)... so actually this would need to be two tests... on the other hand the first one failing convinces me that the second one does as well, so maybe not important enough for stalling this :-D...

Reply:
Actually no: this PR fixes the first failure. It is computed in the `uint` type and thus division is truncating.